src/env.h

```diff
@@ -510,6 +530,7 @@ class Environment {
   AsyncHooks async_hooks_;
   DomainFlag domain_flag_;
   TickInfo tick_info_;
+  ArrayBufferAllocator array_buffer_allocator_;
```

Comment:
don't understand. this class is setup like all the other instance utility classes in `Environment`.